pkg/apis/deployment/v1/tls_sni_spec.go

```diff
@@ -27,6 +27,12 @@ import (
 
 // TLSSNISpec holds TLS SNI additional certificates
 type TLSSNISpec struct {
+	// Mapping is the mapping of the SNI options.
+	// +doc/type: map[string][]string
+	// +doc/example: mapping:
+	// +doc/example:   secret:
+	// +doc/example:   - domain.example.com
```

Comment:
```suggestion
	// +doc/example:     - domain.example.com
```

Reply:
It is valid yaml - list should be indent as the element before